fixtures/chauvet-professional/vesuvio-rgba.json

```
@@ -0,0 +1,699 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Vesuvio RGBA",
+  "shortName": "Chauvet Jet",
+  "categories": ["Smoke", "Color Changer"],
+  "meta": {
+    "authors": ["Beerady", "Luke Nelson"],
+    "createDate": "2021-09-09",
+    "lastModifyDate": "2022-10-17"
+  },
+  "links": {
+    "productPage": [
+      "http://web.archive.org/web/20160421173724/http://www.chauvetprofessional.com/products/vesuvio-rgba/"
+    ],
+    "manual": [
+      "https://www.chauvetprofessional.com/wp-content/uploads/2015/06/Vesuvio-RGBA_UM_Rev4_WO.pdf"
+    ],
+    "video": ["https://www.youtube.com/watch?v=HH5G92l4USw"]
+  },
+  "rdm": {
+    "modelId": 37266,
+    "softwareVersion": "2016"
+  },
+  "physical": {
+    "dimensions": [336, 488, 231],
+    "weight": 13.6,
+    "power": 1590,
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "LED"
+    }
+  },
+  "availableChannels": {
+    "Fog": {
+      "capability": {
+        "type": "Fog",
+        "fogType": "Fog"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightness": "off"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red",
+        "brightness": "off"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green",
+        "brightness": "off"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue",
+        "brightness": "off"
+      }
+    },
+    "Amber": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber",
+        "brightness": "off"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speed": "0Hz"
```

Again, `"speed": "0Hz"` for the whole capability doesn't make sense. The manual mentions `0-100%` again, but that is incomplete as well, since there must be a DMX value (and often it's a range) where Strobe is disabled. So I always add a `helpWanted` message:

```suggestion
        "speedStart": "slow",
        "speedEnd": "fast",
        "helpWanted": "At which DMX values is strobe disabled? When is the light constantly on or off?"
```